<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimCentreDDSClass</class>
 <widget class="QWidget" name="SimCentreDDSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimCentreDDS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>协同控制</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_publisher">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_only_publisher">
          <property name="text">
           <string>只发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>发送数量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_publishernum"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>查询</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_node_search">
             <property name="text">
              <string>查询节点步长指令</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_node_statesearch">
             <property name="text">
              <string>查询节点状态指令</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>步长</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_publisher"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>毫秒</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>协同控制</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="radioButton_direct_push">
               <property name="text">
                <string>直接推进</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_fullspeed_push">
               <property name="text">
                <string>全速推进</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_real_push">
               <property name="text">
                <string>实时推进1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_real_pushtest">
               <property name="text">
                <string>实时推进2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_advance_publisher">
               <property name="text">
                <string>协同推进指令</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_cor_publisher">
               <property name="text">
                <string>协同开始指令</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_pause_publisher">
               <property name="text">
                <string>协同暂停指令</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_stop_publisher">
               <property name="text">
                <string>协同终止指令</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_">
            <property name="title">
             <string>跳转</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>保存节点快照指令</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_recover_snapshoot">
               <property name="text">
                <string>恢复节点快照指令</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QtMyTextEdit" name="textEdit_publisher"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtMyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>QtMyTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SimCentreDDS.qrc"/>
 </resources>
 <connections/>
</ui>
